integrations/maya/src/main/java/org/dash/wallet/integrations/maya/api/ExchangeRateApi.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 Dash Core Group.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.dash.wallet.integrations.maya.api
+
+import org.dash.wallet.integrations.maya.model.ExchangeRateResponse
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+/**
+ * https://exchangerate.host/#/docs
+ */
```

Comment:
This service does not require an API key and supports getting exchange rates in terms of a supported fiat currency such as USD.